### PR TITLE
Story/user research setup

### PR DESCRIPTION
**Pull request by michellexliu**

### CODE REVIEW

File: feedback-module/src/components/Module.js [hunk 1]
```diff
@@ -101,10 +96,20 @@ function Module({ pageTitle, endpoint }) {
...
+        feedback: feedbackForAPI,
+      });
       console.log(feedbackForAPI);
     } else if (formID === "research") {
-      setUserInfo(inputQuestions);
+      let userObj = userInfo;
+      inputQuestions.forEach(({ question, answer }) => {
+        userObj[question] = answer;
+      });
+      userObj.source = window.location.href;
```
Comment (ksiegel1923) [inline feedback-module/src/components/Module.js:109]: Did Justin say that he also wanted the window location to be included in the feedback data as well?

Comment (michellexliu) [reply feedback-module/src/components/Module.js:109]: I believe so— not sure if he meant the full URL or just the path though

Comment (ksiegel1923) [reply feedback-module/src/components/Module.js:109]: So we should probably also add it to the feedback object as a source field right? We can ask Justin if he prefers the full URL or just the path.

Comment (michellexliu) [reply feedback-module/src/components/Module.js:109]: Oh wait sorry I misread you comment— my bad! I don't think he said anything about adding it for the feedback data, just for user research if I remember correctly.

Comment (ksiegel1923) [reply feedback-module/src/components/Module.js:109]: Oh I thought for both but totally could be wrong. Should we check with him?

Comment (michellexliu) [reply feedback-module/src/components/Module.js:109]: Yeah it's def possible I remembered wrong lol I'll check with him

**Pull request merged**